VideoLocker/res/values/strings.xml

```diff
@@ -678,8 +678,11 @@
 
 
     <!-- Discussion/Forum -->
-    <string name="forum_query_no_result">Query <i> \"%1$s\" </i> returns no result.</string>
-    <string name="forum_empty">No result found.</string>
+    <string name="forum_no_results_for_search_query">No results found for \"{search_query}\". Try another search term or select a discussion category to browse.</string>
```

Comment:
> I don't know where I got those fake quotes from.

http://typographyforlawyers.com/straight-and-curly-quotes.html

> Where did you get those quotes from!

This very useful site: http://www.rdquo.net/